packages/grafana-ui/src/options/builder/grid.tsx

```diff
@@ -0,0 +1,23 @@
+import { PanelOptionsEditorBuilder } from '@grafana/data';
+import { VizGridLines, OptionsWithGridLines } from '@grafana/schema';
+
+/**
+ * @public
+ */
+export function addGridLinesOptions<T extends OptionsWithGridLines>(builder: PanelOptionsEditorBuilder<T>) {
+  builder.addRadio({
+    path: 'grid',
+    name: 'Grid lines',
+    category: ['Graph styles'],
```

Comment:
should this not be in the axis category?

Reply:
Asked myself the same question. But decided to put it into graph styles, as it does not affect axis aestethics, but the whole graph.

Reply:
i can see someone wanting the grid to be active on a specific y axis (not always the left one)

Reply:
Yeah, this means this becomes a field config rather than panel option as it is in this PR. Not sure if this is not too much of a flexibility(again). But I see the point for having this config configurable per axis.  The problem with the field config tho is that we currently support a single grid, not per-axis. But, until we do, we can just hide this field config from the overrides, not to introduce confusing parts. 

So, couple of changes:
1. grid option as an axis config
2. We move this under the Axis category and hide from overrides until we decide to support coloring/grid lines per unit


Sounds good gents?

Reply:
:+1: